webapp/packages/plugin-connections/src/ConnectionForm/OriginInfo/OriginInfo.tsx

```diff
@@ -38,12 +40,25 @@ const style = css`
   }
 `;
 
-export const OriginInfo: TabContainerPanelComponent<IConnectionFormProps> = observer(function OriginInfo({ tabId, state: { info, resource } }) {
+export const OriginInfo: TabContainerPanelComponent<IConnectionFormProps> = observer(function OriginInfo({
+  tabId,
+  state: { info, resource, config },
+}) {
   const tab = useTab(tabId);
   const translate = useTranslate();
-  // const userInfoService = useService(UserInfoResource);
+  const userInfoService = useService(UserInfoResource);
```

Comment:
why is it useService instead of useResource?

Reply:
fixed (actually here is both acceptable, but we can modify it to useResource to ensure that user is loaded and up to date in the component scope)